Makefile

```diff
@@ -63,3 +66,22 @@ test:
 
 upload:
 	@NOSCMD=$(NOSCMD) bash build/package/upload.sh $(VERSION)
+
+install-golint: ## check golint if not exist install golint tools
```

Comment:
Please use bash script here.

Reply:
very simple lint check operation, no need to add bash processing, bash is suitable for processing more complex logic

Reply:
> very simple lint check operation, no need to add bash processing, bash is suitable for processing more complex logic

Don't agree. we should keep makefile as simple as possible, so we should put the logic into the bash script. on the other hand, it is more easier to understand and modified if you use bash script, so it's a better choice to use bash syntax, instead of using makefile syntax which has a bit different from the bash syntax.

Reply:
> > very simple lint check operation, no need to add bash processing, bash is suitable for processing more complex logic
> 
> Don't agree. we should keep makefile as simple as possible, so we should put the logic into the bash script. on the other hand, it is more easier to understand and modified if you use bash script, so it's a better choice to use bash syntax, instead of using makefile syntax which has a bit different from the bash syntax.

Just execute a tool command and nothing more